### PR TITLE
W-14476534 Configuring Components doc

### PR DESCRIPTION
**Pull request by chris-campbell9**

### CODE REVIEW

File: modules/ROOT/pages/start-configure-components.adoc [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+====
+Certain component fields can only be configurable via the XML editor.
+====
+
+[NOTE]
+====
+The configuration UI doesn't support configuring components outside of Flow and Subflow components, such as connections and other global configurations.
+====
+
+=== Accessing the Configuration UI
```
Comment (dukesphere) [inline modules/ROOT/pages/start-configure-components.adoc:35]: Adding "UI" and an anchor so I can link to it from the RNs. 

```suggestion
[[config-ui-access]]
=== Accessing the Component Configuration UI
```

File: modules/ROOT/pages/start-configure-components.adoc [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+Anypoint Code Builder enables two ways to access the configuration UI:
+
+* Selecting a component in the canvas.
+* Highlighting a component's XML in the XML editor then selecting *Show Component in Canvas* from the context menu.
+
```
Comment (dukesphere) [inline modules/ROOT/pages/start-configure-components.adoc:41]: not sure that everyone knows what "context menu" means. Can we tell them to put their cursor into the component XML, right click, and select *Show Component in Canvas UI*. 
Screenshot 2024-01-13 at 11 56 59 AM

File: modules/ROOT/pages/start-configure-components.adoc [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+====
+The configuration UI doesn't support configuring components outside of Flow and Subflow components, such as connections and other global configurations.
+====
+
+=== Accessing the Configuration UI
+
+Anypoint Code Builder enables two ways to access the configuration UI:
+
+* Selecting a component in the canvas.
+* Highlighting a component's XML in the XML editor then selecting *Show Component in Canvas* from the context menu.
```
Comment (dukesphere) [inline modules/ROOT/pages/start-configure-components.adoc:40]:
```suggestion
* Highlighting a component's XML in the XML editor then selecting *Show Component in Canvas UI* from the context menu.
```
And pls see comment about "context menu" below.

File: modules/ROOT/pages/start-configure-components.adoc [hunk 4]
```diff
@@ -0,0 +1,130 @@
+= Configuring Components
```
Comment (dukesphere) [inline modules/ROOT/pages/start-configure-components.adoc:1]: Lets change the name of this file to int-configure-components.adoc and move it to the integration section of nav.adoc. The `start`... docs area mainly about things you do or need to know to get up and running, before creating a project. Also, this content is specific to integrations (and implementations), but we've been cover configs in the integration section and reserving the implementation section mostly for scaffolding specs into interfaces that you can then implement.

Comment (chris-campbell9) [reply modules/ROOT/pages/start-configure-components.adoc:1]: Hi @dukesphere, I'm confused. When we last met we discussed **not** publishing this page, but instead rolling my content over into int-create-integrations.adoc (I'm going to add my stuff to that file when you're finished with it.)

Did I misunderstand all this?

**Pull request closed without merging**